src/pages/settings/SettingForm.tsx

```diff
@@ -133,9 +164,19 @@ const SettingForm: FC<Props> = ({ configField, value, isLast }) => {
                   : "You do not have permission to edit server configuration"
               }
             >
-              <div className="readmode-value u-truncate">
-                {getReadModeValue()}
-              </div>
+              {isClustered && isMaasMachine ? (
+                <ClusteredSettingFormInput
```

Comment:
```suggestion
                <ClusteredSettingFormInput
                  key={JSON.stringify(clusteredValue)}
```
Problem with the read not working initially is, that the `clusteredValue` is updated on load from empty object to an object with values. But this component keeps an internal state as the `value` and doesn't reflect updates to the `clusteredValue`. A workaround is to use the json version of `clusteredValue` as a key, to enforce the component to re-initialize. This has the desired side effect of updating the internal state and displaying the correct values.